compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -206,6 +217,12 @@ static ReorderWorkgroupsStrategy getReorderWorkgroupsStrategy(
   return option.value_or(clReorderWorkgroupsStrategy);
 }
 
+// Reconciles log2 of the workgroup reordering tile size based on the pipeline
+// `option` and the CLI flag.
+static unsigned
+getReorderWorkgroupsLogTileSize(const std::optional<int64_t> &option) {
```

Comment:
```suggestion
getReorderWorkgroupsLogTileSize(std::optional<int64_t> option) {
```

This is a very small type, we can pass by value